### PR TITLE
feat: Add intentional panic to check if clusterfuzzlite finds it

### PR DESCRIPTION
**Pull request by netrome**

### CODE REVIEW

File: fuel-vm/src/interpreter/alu.rs [hunk 1]
```diff
@@ -105,14 +105,8 @@ where
...
-    if let Ok(expo) = u32::try_from(c) {
-        Word::overflowing_pow(b, expo)
-    } else if b < 2 {
-        (b, false)
-    } else {
-        (0, true)
-    }
+pub(crate) fn exp(_b: Word, _c: Word) -> (Word, bool) {
+    panic!("Can you find me, mr. Fuzz?");
 }
```
Comment (xgreenx) [inline fuel-vm/src/interpreter/alu.rs:110]: Can we try something like?

```suggestion
     if let Ok(expo) = u32::try_from(c) {
         Word::pow(b, expo).unwrap()
     } else if b < 2 {
         (b, false)
     } else {
         (0, true)
     }
```

**Review by xgreenx — COMMENTED** [review]
Looks like it works=) Jus curious about another use case=)

File: fuel-vm/src/interpreter/alu.rs (hunk 1 above)
Comment (netrome) [reply fuel-vm/src/interpreter/alu.rs:110]: Sure thing, let's try it :)

Comment (netrome) [reply fuel-vm/src/interpreter/alu.rs]: Task failed successfully
image
image

Comment (xgreenx) [reply fuel-vm/src/interpreter/alu.rs]: Cool!=) I think we can close this PR

Comment (netrome) [reply fuel-vm/src/interpreter/alu.rs]: Done

**Pull request closed without merging**